jenkins/worker.py

```diff
@@ -119,6 +137,26 @@ def extract_stats(inp):
   return result
 
 
+def extract_ground_truth_stats(input_text):
+  """Returns a dictionary of ground truth stats."""
+  result = {}
+  parsed = GROUND_TRUTH_STATS_PATTERN.extract(input_text)
```

Comment:
remove empty lines in this function. I python and with the small indent keeping track of blocks is harder and whitespace can hurt readability quite a bit.